README.md

```diff
@@ -1,75 +1,146 @@
-git-crypt - transparent file encryption in git
-==============================================
-
-git-crypt enables transparent encryption and decryption of files in a
-git repository.  Files which you choose to protect are encrypted when
-committed, and decrypted when checked out.  git-crypt lets you freely
-share a repository containing a mix of public and private content.
-git-crypt gracefully degrades, so developers without the secret key can
-still clone and commit to a repository with encrypted files.  This lets
-you store your secret material (such as keys or passwords) in the same
-repository as your code, without requiring you to lock down your entire
-repository.
-
-git-crypt was written by [Andrew Ayer](https://www.agwa.name) (agwa@andrewayer.name).
+# git-crypt - a transparent file encryption in git
+
+<!--- Logo picture element for user's light/dark modes --->
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="./media/git-crypt-logo-dark.svg">
+  <img alt="git-crypt logo header" src="./media/git-crypt-logo.svg">
+</picture>
+
+**git-crypt** enables transparent encryption and decryption of files in your git repository.
+
+Simply select all the files that you want to protect. From that point forward, all your selected files will be encrypted when they are committed and decrypted when they are checked out.
+
+With **git-crypt** you can freely share your repository with the public while keeping your private or sensitive content secure.
+
+**git-crypt** also gracefully degrades, so contributors and developers can still clone and commit changes to your repository while the encrypted files remain secure. Your secret material (such as keys or passwords) can be kept in the same repository as your code, without requiring you to lock down your entire repository.
+
+**git-crypt** was written by [Andrew Ayer](https://www.agwa.name) (agwa@andrewayer.name).
+
 For more information, see <https://www.agwa.name/projects/git-crypt>.
 
-Building git-crypt
-------------------
-See the [INSTALL.md](INSTALL.md) file.
+<!--- Line break elements have been added to improve readability -->
+<br />
+
+## Installing git-crypt
+
+1. For ***nix** based systems.
+
+        apt-get install git-crypt
```

Comment:
`apt` is the package manager in debian based distributions, thus `sudo apt-get install git-crypt` will only work for these distributions. Other operating systems, such as Fedora, Arch, Gentoo, FreeBSD,... have their own package managers. I checked the package repositories for some of the other OSs and most, if not all, provide git-crypt (But via different commands due to different package managers).